mw-webapp/src/service/auth/logIn.ts

```diff
@@ -1,7 +1,7 @@
 import {signInWithRedirect} from "firebase/auth";
 import {auth, provider} from "src/firebase";
 
-export const handleLogIn = async () => {
+export const logIn = async () => {
   try {
     await signInWithRedirect(auth, provider);
```

Comment:
is it possible to rename it to logInWithRedirect?

Reply:
No, this is an internal firebase method. That's why I named function and button value as signIn for the first time.